docker/run_update_data.sh

```diff
@@ -15,6 +15,3 @@ set -eo pipefail
 
 # Insert data that's no longer on archive.mozilla.org
 python ./scripts/insert_missing_versions.py
-
-# Create ES indexes for the next few weeks
-./socorro-cmd create_recent_indices
```

Comment:
Way back in the day my teammate really wanted this line here. I still think it's weird to have most of the "delete and create" things except for ES in `run_setup.sh`. Now that that teammate is gone, I'm moving it.